internal/controllers/core/kubernetesdiscovery/reconciler.go

```diff
@@ -179,6 +189,9 @@ func (w *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 			w.teardown(key)
 			w.cleanupAbandonedNamespaces()
 		}
+
+		// TODO(nick): Delete any orphaned PortForwards or PodLogStreams.
```

Comment:
Ahh whoops, I can't believe I spent the time tracking down the lack of automatic GC on owner references in our apiserver and then didn't actually write the logic to handle it manually 🤦

Reply:
:laughing: